### ISSUE TITLE
[OwnedType] migration is not working as expected

### ISSUE THREAD

**Issue by ShenZZ** [aspnet/EntityFrameworkCore#13661]
[OwnedType] migration is not working as expected

```c#
using Microsoft.EntityFrameworkCore;
using System.Collections.Generic;
public class AppUser
{
    public string Id { get; set; }
    public Address Address { get; set; }
}
public class Order
{
    public string Id { get; set; }
    public List<OrderItem> OrderItem { get; set; }
}
public class OrderItem
{
    public string Id { get; set; }
    public MFile MFile { get; set; }
    public Address Address { get; set; }
}
[Owned]
public class Address
{
    public string Zip { get; set; }
}
[Owned]
public class MFile
{
    public string Uri { get; set; }
}
public class TestDbContext : DbContext
{
    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        optionsBuilder
                .UseSqlServer(@"Server=(localdb)\\mssqllocaldb;Database=Test;");
    }
    public DbSet<AppUser> Users { get; set; }
    public DbSet<Order> Orders { get; set; }
}
public class Program
{
    public static void Main()
    {
        using (var context = new TestDbContext())
        {
            context.Database.EnsureDeleted();
            context.Database.EnsureCreated();
        }
    }
}
```

migration result:
```c#
        protected override void Up(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.CreateTable(
                name: "Orders",
                columns: table => new
                {
                    Id = table.Column<string>(nullable: false)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_Orders", x => x.Id);
                });

            migrationBuilder.CreateTable(
                name: "Users",
                columns: table => new
                {
                    Id = table.Column<string>(nullable: false),
                    Address_Zip = table.Column<string>(nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_Users", x => x.Id);
                });

            migrationBuilder.CreateTable(
                name: "OrderItem",
                columns: table => new
                {
                    Id = table.Column<string>(nullable: false),
                    MFile_Uri = table.Column<string>(nullable: true),
                    Address_Zip = table.Column<string>(nullable: true),
                    Address_OrderItemId = table.Column<string>(nullable: true),  //**this is what?**
                    OrderId = table.Column<string>(nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_OrderItem", x => x.Id);
                    table.ForeignKey(
                        name: "FK_OrderItem_Orders_OrderId",
                        column: x => x.OrderId,
                        principalTable: "Orders",
                        principalColumn: "Id",
                        onDelete: ReferentialAction.Restrict);
                });

            migrationBuilder.CreateIndex(
                name: "IX_OrderItem_OrderId",
                table: "OrderItem",
                column: "OrderId");
        }
```
the problem is "Address_OrderItemId" column in table "OrderItem" ???

### Further technical details
EF Core version: 2.1
Database Provider:  Microsoft.EntityFrameworkCore.SqlServer
Operating system:  Windows 10
IDE:  Visual Studio 2017 15.8.7

**Comment by ajcvickers** [comment]
Note for triage: repros in 2.1.4; fixed in 2.2 preview3:
```C#
columns: table => new
{
    Id = table.Column<string>(nullable: false),
    MFile_Uri = table.Column<string>(nullable: true),
    Address_Zip = table.Column<string>(nullable: true),
    OrderId = table.Column<string>(nullable: true)
},
```

@AndriySvyryd Is this a dupe?

**Comment by AndriySvyryd** [comment]
I don't think we had an issue for this.

**Comment by Eilon** [comment]
Close?

**Comment by ShenZZ** [comment]
Address_Zip = table.Column<string>(nullable: true),
Address_OrderItemId = table.Column<string>(nullable: true),   <== why is this column added ???

**Comment by ajcvickers** [comment]
@ShenZZ It's a bug in 2.1.x, fixed for 2.2.0.
